### PR TITLE
run rustfmt on typecheck/coherence folder

### PR DESCRIPTION
**Pull request by srinivasreddy**

### CODE REVIEW

File: src/librustc_typeck/coherence/mod.rs [hunk 1]
```diff
@@ -65,37 +65,39 @@ impl<'a, 'gcx, 'tcx, 'v> intravisit::Visitor<'v> for CoherenceCheckVisitor<'a, '
...
+            TyArray(..) |
+            TySlice(..) |
+            TyFnDef(..) |
+            TyFnPtr(_) |
+            TyTuple(..) |
+            TyParam(..) |
+            TyError |
+            TyRawPtr(_) |
+            TyRef(_, _) |
+            TyProjection(..) => None,
```
Comment (gereeter) [inline src/librustc_typeck/coherence/mod.rs]: I think this long list is much better compressed as it was before.

Comment (GuillaumeGomez) [inline src/librustc_typeck/coherence/mod.rs]: I have mixed feelings about this. This syntax is more clear and less clear. :-/

Comment (srinivasreddy) [inline src/librustc_typeck/coherence/mod.rs]: cc @nrc

**Comment by alexcrichton** [comment]
Closing due to inactivity, but feel free to reopen with comments addressed!

**Pull request closed without merging**